app/controllers/organisations_controller.rb

```diff
@@ -69,10 +69,12 @@ def create
        redirect_to organisations_path and return false
      end
     @organisation = Organisation.new(org_params)
+    @categories_start_with = Category.first_category_name_in_each_type
```

Comment:
interesting - so the change is in 'create' - we were looking at the update method before.  Not sure why ... ah, validations failing during the creation process rather than the editing process - makes sense.

Hmm, so do we need acceptance tests checking that editing works correctly ... maybe we already have them ...